src/components/homepage.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import AllSports from './allSports';
+import Sport from './sport';
+
+const HomePage = () => (
+  <div>
+    <Routes>
+      {['/', '/react-redux-capstone'].map((path) => <Route path={path} element={<AllSports />} key="key-1" />)}
+      <Route path="/*" element={<Sport />} />
```

Comment:
- [ ] <b>REQUIRED:</b>  You have two routes `['/', '/react-redux-capstone']` that are rendering the same component `<AllSports />`.  the project requires to have at least one route to render the list of items and another one to render the details of a single item. Besides, the is no point in having two routes rendering the same component when there is no `redirect` functionality.  Kindly address it and fix it.